go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -36,7 +36,7 @@ type nodeNetworkControllerManager struct {
 	wg            *sync.WaitGroup
 	recorder      record.EventRecorder
 
-	defaultNodeNetworkController nad.BaseNetworkController
```

Comment:
This is the bit that I didn't question in my PR and which made me add `preStart` to the interface. My bad! Thanks for improving this!

Reply:
same here nice we changed this.. when I added the dereference I wanted to come back to this to find out why DNNC had nad.BNC type...thanks for changing this